lib/common.sh

```diff
@@ -90,20 +90,52 @@ export SSH_PUB_KEY_CONTENT
 FILESYSTEM=${FILESYSTEM:="/"}
 
 # Environment variables
-# M3PATH : Path to clone the Metal3 Development Environment repository
-# BMOPATH : Path to clone the Bare Metal Operator repository
+# M3PATH    : Path to clone the Metal3 Development Environment repository
+# BMOPATH   : Path to clone the Bare Metal Operator repository
 # CAPM3PATH : Path to clone the Cluster API Provider Metal3 repository
-# CAPIPATH : Path to clone the Cluster API repository
+# CAPIPATH  : Path to clone the Cluster API repository
+# IPAMPATH  : Path to clone Ip Address Manager
+#
+# CAPIREPO    : Cluster API git repository URL
+# CAPIBRANCH  : Cluster API git repository branch to checkout
+# CAPICOMMIT  : Cluster API git commit to check out on CAPIBRANCH
+# BMOREPO     : Baremetal Operator git repository URL
+# BMOBRANCH   : Baremetal Operator git repository branch to checkout (its set it lib/releases.sh)
+# BMOCOMMIT   : BMO git commit to check out on BMOBRANCH
+# CAPM3REPO   : Cluster API Provider Metal3 git repository URL
+# CAPM3BRANCH : Cluster API Provider Metal3 git repository branch to checkout
+# CAPM3COMMIT : CAPM3 git commit to check out on CAPM3BRANCH
+# IPAMREPO    : IP Address Manager git repository URL
+# IPAMBRANCH  : IP Address Manager git repository branch to checkout
+# IPAMCOMMIT  : IP Address Manager git commit to check out on IPAMBRANCH
```

Comment:
maybe we should stick to one: `checkout` or `check out` everywhere (first looks better IMO)